lib/taskana-core/src/test/java/acceptance/task/UpdateTaskAccTest.java

```diff
@@ -129,6 +130,31 @@ void should_ThrowException_When_TaskHasAlreadyBeenUpdated()
         .withFailMessage("The task has already been updated by another user");
   }
 
+  @WithAccessId(user = "admin")
+  @WithAccessId(user = "taskadmin")
+  @TestTemplate
+  void should_UpdateTask_When_NoExplicitPermissionsButUserIsInAdministrativeRole()
+      throws TaskNotFoundException, ClassificationNotFoundException, InvalidArgumentException,
+                 ConcurrencyException, NotAuthorizedException, AttachmentPersistenceException,
+                 InvalidStateException, SQLException {
+
+    TaskService taskService = taskanaEngine.getTaskService();
+    Task task = taskService.getTask("TKI:000000000000000000000000000000000000");
+    final ClassificationSummary classificationSummary = task.getClassificationSummary();
+    task.setClassificationKey("T2100");
+    Task updatedTask = taskService.updateTask(task);
+    updatedTask = taskService.getTask(updatedTask.getId());
+
+    assertThat(updatedTask).isNotNull();
```

Comment:
in this testcase we do not need all the assertions, do we? I assume we have this tested in another testcase?

Reply:
changed it